### DISCUSSION TITLE
Deepmd plus lammps md simulations

### DISCUSSION THREAD

**Discussion by lanjh09** [deepmodeling/deepmd-kit#2247] [Q&A]
I have tried running md simulation of organic molecules in water based on the deepmd train. Howvever, the structure of organiic molecules  broke down at the beginning. Could anyone give me some suggestions? 

The system contains 3 organic molecules and water as well as a metal ion.
The input is like this:

{
    "_comment": " model parameters",
    "model": {
        "type_map":     ["C","H","N","O","U"],
        "descriptor" :{
            "type":             "se_e2_a",
            "sel":              [46, 92, 46, 46, 46],
            "rcut_smth":        0.50,
            "rcut":             6.00,
            "neuron":           [25, 50, 100],
            "resnet_dt":        false,
            "axis_neuron":      16,
            "seed":             1,
            "_comment":         " that's all"
        },
        "fitting_net" : {
            "neuron":           [240, 240, 240],
            "resnet_dt":        true,
            "seed":             1,
            "_comment":         " that's all"
        },
        "_comment":     " that's all"
    },

       "learning_rate" :{
        "type":         "exp",
        "decay_steps":  50,
        "start_lr":     0.001,
        "stop_lr":      3.51e-8,
        "_comment":     "that's all"
    },

    "loss" :{
        "type":         "ener",
        "start_pref_e": 0.02,
        "limit_pref_e": 1,
        "start_pref_f": 1000,
        "limit_pref_f": 1,
        "start_pref_v": 0,
        "limit_pref_v": 0,
        "_comment":     " that's all"
    },

**Comment by njzjz** [comment]
Please read the [checklist in the documentation](https://docs.deepmodeling.com/projects/deepmd/en/v2.2.0.b0/troubleshooting/precision.html) to see if anything can be improved. Thanks!
